plugins/auth-backend-module-okta-provider/README.md

```diff
@@ -0,0 +1,8 @@
+# Auth Module: Okta Provider
+
+This module provides an Okta auth provider implementation for `@backstage/plugin-auth-backend`.
```

Comment:
Suggestion: I think adopters would appreciate more documentation about the configuration options, as well as an example of how to install this module.
What are your thoughts on this?

Reply:
@camilaibs I agree, and I have included an example of how to use this module in the readME.